html-css/portfolio.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Welcome to @Amiinata's Portfolio</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="#about">About</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#projects">Projects & Achievements </a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+    <header>
+        <img src="IMG_8571.jpg" alt=" ">
+        <h1> Aminata BA </h1>
+        <p> 22  |  Web Development & Design Enthusiast | HackyourFuture30' student</p>
+    </header>
+
+    <div class="about-me">
+        <h2>Who am I ? </h2>
+        <p> A wannabe engineer from Senegal currently based in Denmark,  and a web development enthusias, with a focus on designing and creating clean, responsive, and user-friendly websites.</p>
+    </div>
+<div class="skills">
+        <h3>  Skills </h3>
+        <p> Here is a list of what i can do </p>
+</div>
+        <div class="skills-container">
+            <div class="skill">HTML</div>
+            <div class="skill">CSS</div>
+            <div class="skill"> (A little bit of) JavaScript </div>
+        
+        </div>
+    </div>
+
+    <div class=" Projects ">
```

Comment:
There's an unnecessary space in the class name " Projects " For consistency, it's nice to have all classes in lowercase either with hyphens or without so it is better to change the "P" in "Project" to a lowercase to have a cleaner code.